### PR TITLE
Added Listof component

### PR DESCRIPTION
**Pull request by Tramsi**

### CODE REVIEW

File: src/components/shared/ListOf/index.jsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            {timeElapsed(createdAt)}
+          </p>
+          <p className="font-bold text-c100 leading-relaxed">{description}</p>
+        </div>
+      </article>
+    </animated.a>
+  );
+};
+
+const ListOf = ({ data, loading, error }) => {
```
Comment (Nilegfx) [inline src/components/shared/ListOf/index.jsx:53]: Too generic name for very specific component!

Comment (Tramsi) [reply src/components/shared/ListOf/index.jsx:53]: I know :/ .... I spent a lot of time trying to find a suitable name and couldn't find a proper one

File: src/components/RecentArticles/index.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import { ListItemWithThumbnail } from '../shared/ListItemWithThumbnail';
+
+const RecentArticles = ({ data, loading, error }) => {
+  const isMobile = useMedia(['(min-width: 768px)'], [false], true);
+
+  if (error) {
+    return "Couldn't fetch data";
+  }
+
+  let mobileLoaders = [];
```
Comment (Nilegfx) [inline src/components/RecentArticles/index.jsx:15]: better to move it inside `if(loading)` condition

File: src/components/RecentArticles/index.jsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            return <ListItemWithThumbnail data={articleData} />;
+          })}
+        </div>
+      </Widget>
+    );
+  }
+};
+
+const RecentArticlesContainer = () => {
+  const { data, loading, dataError: error } = useCharityAPI('/supporters');
```
Comment (Nilegfx) [inline src/components/RecentArticles/index.jsx:57]: haza3alak ya Tramsi :D

File: src/components/RecentArticles/myLoader.jsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    speed={2}
+    viewBox="0 0 110 30"
+    backgroundColor="#f3f3f3"
+    foregroundColor="#ecebeb"
+  >
+    <rect x="0" y="0" rx="15" ry="15" width="110" height="30" />
+  </ContentLoader>
+);
+
+export const ButtonLoader = () => (
```
Comment (Nilegfx) [inline src/components/RecentArticles/myLoader.jsx:19]: please call it `chevron`

**Comment by Tramsi** [comment]
> LGTM
> Happy Eid :))

TYYYY!!!!

**Pull request merged**